policyengine_us/variables/gov/states/nc/tax/income/nc_use_tax.py

```diff
@@ -0,0 +1,21 @@
+from policyengine_us.model_api import *
+
+
+class nc_use_tax(Variable):
```

Comment:
add this variable to the nc_income_tax var, see california use tax as an example